api/api/spec/spec.yaml

```diff
@@ -14383,6 +14408,51 @@ paths:
                 message: "All specified task's status were returned"
                 error: 0
 
+  /content/update:
+    put:
+      tags:
+        - Content
+      summary: "Update content on on-demand"
+      description: "Updates the Engine and VD content on all nodes in the cluster."
+      operationId: api.controllers.content_controller.put_content_update
+      x-rbac-actions:
+        - $ref: '#/x-rbac-catalog/actions/security:update'
```

Comment:
This should be changed in favor of a new action/ resource or use a similar like analysisd reload